api_contracts/contracts/award/AwardProfile.md

```diff
@@ -432,4 +472,38 @@ This endpoint returns aggregated award amounts for IDVs.
 + contract_count: 0 (required, number)
 + rollup_base_exercised_options_val: 0.00 (required, number)
 + rollup_base_and_all_options_value: 106321.10 (required, number)
-+ rollup_total_obligation: 106321.10 (required, number)
\ No newline at end of file
++ rollup_total_obligation: 106321.10 (required, number)
+
+## IDVRelatedAwardsResponse (object)
++ award_id: 12178065 (required, number)
+    The award id sent in the request.
++ generated_unique_award_id: `CONT_AW_1540_NONE_DJB30605051_NONE` (required, string)
++ piid: `W31P4Q15A0024` (required, string)
++ funding_agency (required, Agency, fixed-type)
++ award_type: `BPA`(required, string)
++ obligated_amount: 242342 (required, number)
++ description: `reerwrawa` (required, string, nullable)
++ period_of_performance_start_date: `2004-02-19` (required, string, nullable)
+    The starting date of the contract in the format `YYYY-MM-DD`
++ period_of_performance_current_end_date: `2005-02-19` (required, string, nullable)
+    The ending date of the contract in the format `YYYY-MM-DD`
++ last_date_to_order: `2301-02-23` (required, string, nullable)
+
+## PageMetaDataObject (object)
```

Comment:
PageMetaDataObject, Agency, TopTierAgency and Subtier Agency have all been defined earlier in the contract. Could these duplicate entries be removed?